package-lock.json

```diff
@@ -64992,6 +64992,7 @@
 		"packages/discussions": {
 			"name": "@esri/hub-discussions",
 			"version": "25.8.0",
+			"version": "25.8.0",
```

Comment:
Hey heads up that this line got duplicated, you'll want to remove it before we merge.